### ISSUE TITLE
Users need to upload a photograph but were discouraged from the start and weren't sure they'd be able to do so

### ISSUE THREAD

**Issue by irinapencheva** [alphagov/verify-local-patterns#458]
Those that were aware of the need to upload a photo were discouraged by it.

Users did not think this would be something they would be able to do unassisted and may seek another channel to complete the transaction.

**Comment by suryanaga** [comment]
@irinapencheva @petegale 

I'd like to get clarity on where in the journey we think this drop off happens for most users. 

Do people need more reassurance on the start page that we'll help them uploading the photo?

Or do they not realise they'll need a photo until the start of the upload process, and then effectively drop off there?

Note: Let's continue to be clear that this issue is about the user being *confident that they will be able to succeed* as opposed to *actually succeeding*, which is more what issue #347 is about.

**Comment by irinapencheva** [comment]
@sanjaypoyzer - this  is about reassuring people from the start that we'll help and they'll probably be able to upload a photo

The issue about them not realising they'll need a photo until the start of the upload process is hopefully addressed now as part of the reworded "Verify your identity page"

**Comment by lizziebruce** [comment]
@irinapencheva @sanjaypoyzer 

Content fix:

> 2. Upload a clear digital photo of yourself by following our instructions.

Below is the content fix in context of other fixes for the Start page from my [CD review](https://docs.google.com/a/digital.cabinet-office.gov.uk/document/d/1e9XkmzaWnWnm9Xa21vovx7Tm-p7shateiultu78ZBBA/edit?usp=sharing)  

But perhaps we should also consider putting About your application content on its own on the next page, so that people are more likely to read it.

> # Apply for an older person’s bus pass
> 
> Older [residents of Argleton](https://www.gov.uk/find-local-council) can get a free bus pass. When you can apply depends on when you were born. 
> 
> Enter your date of birth 
> [  widget  ] 
> [ Yes, you can apply now   [[OR]]  You can’t apply until 2020]
> 
> The pass is valid: 
> 
> - on all local bus services in England 
> - 9:30am to 11pm weekdays
> - anytime Saturday, Sunday and bank holidays
> 
> ## About your application
> 1. Identify yourself using GOV.UK Verify, or by scanning documents.
> 2. Upload a clear digital photo of yourself by following our instructions.
> 3. Submit your application, we’ll send your bus pass by post.
> 
> ## Apply for an older person’s bus pass >>
> 
> You can also:
> 
> - replace a lost or stolen bus pass
> - renew an expired bus pass
> 
> If you need assistance using the online service, contact Argleton County Council
> 
> **Other ways to apply**

**Comment by suryanaga** [comment]
@lizziebruce Yeah! I really like that line and I think it will help. I was trying to think of something like that but couldn't.

**Comment by irinapencheva** [comment]
@lizziebruce - I also really like pt 2 (maybe just removing the word 'clear' as discussed).

I also would like to test splitting the page into 2 and see whether users read 'About your application' section if it's on a separate page.

**Comment by lizziebruce** [comment]
@irinapencheva @sanjaypoyzer new copy suggestion for About your application section following discussion about how Verify checks aren't always questions.

## About your application

1. Complete some online security checks or scan documents to show you’re eligible.
2. Upload a digital photo – we'll give you guidance on doing this.
3. Submit your application online.
4. We'll send your bus pass to you by post.

**Comment by lizziebruce** [comment]
@irinapencheva please see Pull Request #521

**Comment by irinapencheva** [comment]
@lizziebruce - thanks, I've approved it in the pull request

**Comment by petegale** [comment]
Limited evidence (2 users) from Round 3 suggests this may have been resolved, however not conclusive yet so keeping open

**Comment by lizziebruce** [comment]
@irinapencheva @petegale based on the R3 research:

'The title of the skipped section does not contain task specific keywords so is more likely to be ignored.'

I would like to reword:

> About your application 

to 

> How to apply 

as

'Users may not be aware of the need to upload a photograph later in the application and may start without being able to complete this step.'

**Comment by irinapencheva** [comment]
@lizziebruce - yes, agreed, go for it!
